README.md

```diff
@@ -118,6 +128,8 @@ version for .deb artifact will be constructed based on current time. Pattern ```
 	<configuration>
 		<unixUserId>ubuntu</unixUserId>
 		<unixGroupId>ubuntu</unixGroupId>
+		<installDir>/usr/bin</installDir>
+		<appShortcutsSourceDir>${basedir}/src/shortcuts</appShortcutsSourceDir>
```

Comment:
This sounds very specific. Can you extract into app-specific postinst? I
put it into: ${basedir}/src/deb/postinst:

cp ${config.installDir}/etc/myapp-prod.conf /etc/nginx/sites-enabled/myapp;

This will copy my app config (or folder in your case) to the proper directory.

Reply:
I guess this is the last 👍

Reply:
You're right, even though it would be useful for every desktop application, it is rather a convenience function for a task which can also be accomplished in a generic way via postins/postrm. I'll remove that part as soon as I get to it. (probably tomorrow)

Reply:
No worries. You've already did great contribution! Thank you.

Reply:
Reverted. I did some testing with `https://github.com/retgal/dayon/` and subsequently enhanced the post rm/inst a bit.